kore/src/Kore/Repl/Data.hs

```diff
@@ -825,3 +856,118 @@ substituteAlias
     toString = \case
         SimpleArgument str -> str
         QuotedArgument str -> "\"" <> str <> "\""
+
+createOnePathClaim
+    :: Claim claim
+    => (claim, CommonStrategyPattern)
+    -> Rule.OnePathRule Variable
+createOnePathClaim (cl, cpatt) =
```

Comment:
Maybe not as verbose as this article proposes (https://chrisdone.com/posts/german-naming-convention/), but in this case `claim` and `pattern` would make better, clearer names.